File: bank/BankAccount.php
<?php

class BankAccount
{
    public function __construct(
        public string $owner,
        private float $balance
    ) {
    }

    public function deposit(float $amount)
    {
        if ($amount <= 0) {
            echo "depozītam jābūt pozitīvam {$this->owner}.<br>\n";
            return;
        }

        $this -> balance + $amount;
        echo "Ielikts {$amount} iekšā {$this->owner}.<br>\n";

        
    }
    public function withdraw(float $amount)
    {
        if ($amount <= 0) {
            echo "Summai jābūt pozitīvam {$this->owner}.<br>\n";
            return;
        }
         if ($amount > $this->balance) {
            echo "Nepietiekami līdzekļi {$this->owner}.<br>\n";
            return;
        }
        $this->balance -= $amount;
        echo "Izņemts {$amount} no {$this->owner}.<br>\n";
    }
    public function printInfo(): void
    {
        $balanceFormatted = number_format($this->balance, 2);
        echo "Owner: {$this->owner} | Balance: {$balanceFormatted}.<br>\n";
    }
}

File: python bank/BankAccount.py
class BankAccount:
    def __init__(self, owner: str, balance: float):
        self.owner = owner
        self.balance = float(balance)

    def deposit(self, amount: float):
        if amount <= 0:
            print(f"depozītam jābūt pozitīvam {self.owner}.")
            return
        self.balance += amount
        print(f"Ielikts {amount} iekšā {self.owner}.")

    def withdraw(self, amount: float):
        if amount <= 0:
            print(f"Summai jābūt pozitīvam {self.owner}.")
            return
        if amount > self.balance:
            print(f"Nepietiekami līdzekļi {self.owner}.")
            return
        self.balance -= amount
        print(f"Izņemts {amount} no {self.owner}.")

    def print_info(self):
        balance_formatted = f"{self.balance:,.2f}"
        print(f"Owner: {self.owner} | Balance: {balance_formatted}.")



